netctl/commands.go

```diff
@@ -450,6 +450,56 @@ var Commands = []cli.Command{
 		},
 	},
 	{
+		Name:  "aci-gw",
+		Usage: "ACI Gateway information",
+		Subcommands: []cli.Command{
+			{
+				Name:      "info",
+				Usage:     "Show aci gateway information",
+				ArgsUsage: " ",
+				Flags:     []cli.Flag{tenantFlag, allFlag, jsonFlag},
+				Action:    showAciGw,
+			},
+			{
+				Name:      "inspect",
+				Usage:     "Inspect aci gateway operational information",
+				ArgsUsage: " ",
+				Flags:     []cli.Flag{jsonFlag},
+				Action:    inspectAciGw,
+			},
+			{
+				Name:      "set",
+				Usage:     "Set aci-gw parameters",
+				ArgsUsage: " ",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "path-bindings, p",
+						Usage: "Blank or  Comma separated paths of the form topology/pod-1/paths-101/pathep-[eth1/14]",
+					},
+					cli.StringFlag{
+						Name:  "node-bindings, n",
+						Usage: "Blank or  Comma separated nodes of the form topology/pod-1/node-101",
+					},
+					cli.StringFlag{
+						Name:  "phys-dom, d",
+						Usage: "ACI physical domain name (e.g. containerDom)",
+					},
+					cli.StringFlag{
+						Name:  "enforce-policies, e",
```

Comment:
This is one of the existing env vars. The second one is to utilize the common tenant in aci.